service/application/src/main/java/com/solace/maas/ep/common/model/ResourceConfigurationType.java

```diff
@@ -0,0 +1,6 @@
+package com.solace.maas.ep.common.model;
+
+public enum ResourceConfigurationType {
+    SOLACE,
+    KAFKA,
```

Comment:
Since we haven't added CONFLUENT_SCHEMA_REGISTRY here, I suggest removing `KAFKA` for the time being until we are ready to support that. Will be an easy update.